### DISCUSSION TITLE
Sequential Merge Requests

### DISCUSSION THREAD

**Discussion by cbn-targit** [renovatebot/renovate#27932] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure DevOps, using renovate@37.244.1 CLI

### Please tell us more about your question or problem

Is it possible, using renovate, to make "sequential merges" (Lack of better phrase)

The effect i would like is for Renovate to be able to do the following
(Renovate Branch) -> Develop --- (Once merged, create new PR) --> Staging --- (Once merged, create new PR) ---> Production

Is this possible to do? I know i can "just" have multiple base branches, but that messes up the Git history

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment]
no

**Comment by jamietanna** [comment, marked as answer]
Yes: https://www.jvt.me/posts/2026/09/08/renovate-staged-branches/
